specification/storageactions/resource-manager/Microsoft.StorageActions/stable/2023-01-01/storageactions.json

```diff
@@ -556,12 +556,63 @@
         },
         "properties": {
           "x-ms-client-flatten": true,
-          "$ref": "#/definitions/StorageTaskProperties",
+          "$ref": "#/definitions/StorageTaskUpdateProperties",
           "description": "Properties of the storage task."
         }
       },
       "description": "Parameters of the storage task update request"
     },
+    "StorageTaskUpdateProperties": {
```

Comment:
I believe this change don't need be included in 4/24 SDK release, please confirm.

Besides that, could you share the reason to split "StorageTaskUpdateProperties" from  "StorageTaskProperties"?

Reply:
@blueww the swagger was wrong here, PATCH requests don't need to have all these required properties. They are only required for PUT requests.